lumicks/pylake/force_calibration/calibration_item.py

```diff
@@ -130,6 +160,43 @@ def __repr__(self):
         )
         return f"{self.__class__.__name__}({properties})"
 
+    def _check_has_data(self):
+        if not self.voltage:
```

Comment:
again here might be safer to do `if self.voltage is None`

Reply:
Here the intent was falsey. By default construction, this would be an `empty_slice`. If you think it is clearer, I can change it to `not len(voltage)`.